var/spack/repos/spack_repo/builtin/packages/quandary/package.py

```diff
@@ -0,0 +1,64 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack_repo.builtin.build_systems.cached_cmake import CachedCMakePackage
+from spack_repo.builtin.build_systems.cuda import CudaPackage
+from spack_repo.builtin.build_systems.rocm import ROCmPackage
+
+from spack.package import *
+
+
+class Quandary(CachedCMakePackage, CudaPackage, ROCmPackage):
+    """Optimal control for open quantum systems"""
+
+    homepage = "https://github.com/LLNL/quandary"
+    git = "https://github.com/LLNL/quandary.git"
+
+    maintainers("steffi7574", "tdrwenski")
+
+    license("MIT", checked_by="tdrwenski")
+
+    version("main", branch="main", preferred=True)
+    version("learning", branch="learning")
+
+    variant("slepc", default=False, description="Build with Slepc library")
+    variant("debug", default=False, description="Debug mode")
+    variant("test", default=False, description="Add dependencies needed for testing")
+
+    depends_on("cxx", type="build")
+
+    depends_on("petsc~hypre~metis~fortran")
+    depends_on("petsc+debug", when="+debug")
+    depends_on("slepc", when="+slepc")
+
+    depends_on("blt", type="build")
```

Comment:
Could you please double check that you do not depend on a BLT newer than that?

Reply:
Thanks! just tested and it works with 0.6.0 and 0.7.0 so and using `0.6.0:` in 6e6818ebd3463a70b1ff91b9dfade7972f2e9373